### ISSUE TITLE
Chapter: About this book

### ISSUE THREAD

**Issue by rauschma** [rauschma/exploring-reasonml#1]

**Comment by sunithex** [comment]
Hi, just chanced upon your book. Could you please let me know if its been completed?

**Comment by rauschma** [comment]
@sunithex Yes it is finished and, sadly, outdated by now. I’ve added a warning here: http://reasonmlhub.com/exploring-reasonml/ch_about-book.html#warning-this-book-is-outdated

**Comment by fcidade** [comment]
@rauschma Is there a way we can contribute by updating the book to the newer versions and/or translating to other languages?
